src/System.ComponentModel.Composition/src/System.ComponentModel.Composition.csproj

```diff
@@ -185,18 +190,13 @@
     <Compile Include="System\ComponentModel\Composition\ReflectionModel\ReflectionProperty.cs" />
     <Compile Include="System\ComponentModel\Composition\ReflectionModel\ReflectionType.cs" />
     <Compile Include="System\ComponentModel\Composition\ReflectionModel\ReflectionWritableMember.cs" />
+    <Compile Include="System\Strings.Designer.cs" />
     <Compile Include="$(CommonPath)\Microsoft\Internal\Assumes.cs">
       <Link>Microsoft\Internal\Assumes.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Microsoft\Internal\Assumes.InternalErrorException.cs">
       <Link>Microsoft\Internal\Assumes.InternalErrorException.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Microsoft\Internal\CommonStrings.Designer.cs">
```

Comment:
I would suspect this to be necessary so that you don't change the common Assumes.InternalException files which are used by other csproj files, as this are shared resources.